liteflow-labs/starter-kit
# Accept partial fill for bids

### Description

Add a modal when we accept a bid with multiple edition displaying a form to select the quantity we want to accept

This allows having partial fill even on the bids.

### How to test

Place a bid requesting multiple editions on an ERC1155, then connect to an account(s) that can fill this bid (even partially), then accept the bid. You should have a popup with a form to let you choose how much you want to accept
